save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/test/PublicTestDto.kt

```diff
@@ -0,0 +1,19 @@
+/**
+ * DTOs for public test transfer
+ */
+
+package com.saveourtool.save.test
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property testLines public test's test file lines
+ * @property expectedLines public test's expected file lines
+ * @property tags list of tags of current test
+ */
+@Serializable
+data class PublicTestDto(
```

Comment:
I don't think it should be called `public`, because for example if we implement a view for manaing contests by admins, they also will be able to see tests from this structure. I think it can be called `TestFilesContent` similar to `Plugin.TestFiles`

Reply:
Makes sense